spec/javascripts/LoginPage/__snapshots__/LoginPageWrapper.spec.tsx.snap

```diff
@@ -997,12 +996,11 @@ exports[`should render Login form and Authentication providers when available 1`
                                     "verticalAlign": "-0.125em",
                                   }
                                 }
-                                viewBox="0 0 512 512"
+                                viewBox="0 0 1024 1024"
                                 width="1em"
                               >
                                 <path
-                                  d="M512 176.001C512 273.203 433.202 352 336 352c-11.22 0-22.19-1.062-32.827-3.069l-24.012 27.014A23.999 23.999 0 0 1 261.223 384H224v40c0 13.255-10.745 24-24 24h-40v40c0 13.255-10.745 24-24 24H24c-13.255 0-24-10.745-24-24v-78.059c0-6.365 2.529-12.47 7.029-16.971l161.802-161.802C163.108 213.814 160 195.271 160 176 160 78.798 238.797.001 335.999 0 433.488-.001 512 78.511 512 176.001zM336 128c0 26.51 21.49 48 48 48s48-21.49 48-48-21.49-48-48-48-48 21.49-48 48z"
-                                  transform=""
+                                  d="M802,320 C748.980664,320 706,277.019336 706,224 C706,170.980664 748.980664,128 802,128 C855.019336,128 898,170.980664 898,224 C898,277.019336 855.019336,320 802,320 M704,0 C527.3,0 384,143.3 384,320 C383.937788,357.490503 390.505571,394.696657 403.4,429.9 L0,824.1 L0,1024 L192,1024 L192,896 L320,896 L320,768 L448,768 L597,622 C596.906403,621.881923 596.838304,621.745723 596.8,621.6 C631.220126,633.811107 667.47802,640.034477 704,640 C880.7,640 1024,496.7 1024,320 C1024,143.3 880.7,0 704,0"
                                 />
```

Comment:
this reminds me of SVG

Reply:
These are the icons being updated because @patternfly/react-core depends on a more recent version.
I will update the message.